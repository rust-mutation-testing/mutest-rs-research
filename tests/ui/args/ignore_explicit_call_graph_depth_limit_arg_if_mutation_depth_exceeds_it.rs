//@ build
//@ stderr
//@ mutest-flags: --call-graph-depth-limit=3 --depth=5
