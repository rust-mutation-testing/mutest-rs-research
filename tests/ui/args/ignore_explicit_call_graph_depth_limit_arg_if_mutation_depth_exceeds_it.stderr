warning: explicit call graph depth limit argument ignored as mutation depth exceeds it
  |
  = note: mutation depth is set to 5
  = note: call graph depth limit was explicitly set to 3, but will be ignored

